ports/esp32s2/common-hal/touchio/TouchIn.c

```diff
@@ -25,10 +25,12 @@
  */
 
 #include "shared-bindings/touchio/TouchIn.h"
-#include "py/runtime.h"
 
+#include "py/runtime.h"
 #include "driver/touch_pad.h"
 
+bool touch_inited = false;
```

Comment:
Want to reset this every VM run? That way it will be shutdown if no longer needed.